ui/page/components/components.go

```diff
@@ -117,110 +116,68 @@ func LayoutBalance(gtx layout.Context, l *load.Load, amount string) layout.Dimen
 // direction, balance, status.
 func LayoutTransactionRow(gtx layout.Context, l *load.Load, row TransactionRow) layout.Dimensions {
 	gtx.Constraints.Min.X = gtx.Constraints.Max.X
-	directionIconTopMargin := values.MarginPadding16
-
-	if row.Index == 0 && row.ShowBadge {
-		directionIconTopMargin = values.MarginPadding14
-	} else if row.Index == 0 {
-		// todo: remove top margin from container
-		directionIconTopMargin = values.MarginPadding0
-	}
 
 	wal := l.WL.MultiWallet.WalletWithID(row.Transaction.WalletID)
 
-	return layout.Inset{Top: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
-		return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+	return layout.Inset{Top: values.MarginPadding16, Bottom: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
```

Comment:
The padding used on the recent transaction page is not `values.MarginPadding16` its `values.MarginPadding5`
<img width="473" alt="image" src="https://user-images.githubusercontent.com/27733432/130025782-0fd62f17-cb0c-436c-935e-ae3f9edecbba.png">

this is the reason we currently have excess padding on the recent transactions page.